src/main/java/rx/Completable.java

```diff
@@ -1127,6 +1132,7 @@ public final Completable compose(CompletableTransformer transformer) {
      *  <dd>{@code andThen} does not operate by default on a particular {@link Scheduler}.</dd>
      * </dl>
      *
+     * @param <T> the value type of the next Observable
```

Comment:
Thanks. Once I'm done with the rearchitecting in the 2.x branch, I'll update the PR.